src/main/kotlin/no/nav/medlemskap/services/oppgave/OppgaveClient.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.medlemskap.services.oppgave
+
+import io.ktor.client.request.get
+import io.ktor.client.request.header
+import io.ktor.client.request.parameter
+import io.ktor.client.request.url
+import io.ktor.http.HttpHeaders
+import no.nav.medlemskap.common.defaultHttpClient
+import no.nav.medlemskap.modell.oppgave.FinnOppgaverResponse
+import no.nav.medlemskap.services.sts.StsRestClient
+
+private const val TEMA_MEDLEMSKAP = "MED"
```

Comment:
Fin bruk av konstanter for å gjøre det mer forståelig, det burde jeg gjøre i Saf-koden også!